tsconfig.json

```diff
@@ -0,0 +1,25 @@
+{
+  "compilerOptions": {
+    "module": "CommonJS",
+    "lib": [
+      "esnext"
+    ],
+    "target": "ES6",
```

Comment:
Change target to be at least ES2017, or higher. With ES6, it's polyfilling Promises which isn't necessary.
```suggestion
    "target": "ES2017",
```

Reply:
Seems to compile/test with ESNext, does that sound good?

Reply:
Seems to compile/test with ESNext. Does that sound good?